src/sonic-yang-mgmt/yang-models/sonic-acl.yang

```diff
@@ -170,7 +170,7 @@ module sonic-acl {
 				choice icmp {
 
 					case icmp4 {
-						when "boolean(IP_TYPE[.='ANY' or .='IP' or .='IPV4' or .='IPV4ANY' or .='ARP'])";
+						when "boolean(IP_TYPE[.='ANY' or .='IP' or .='IPV4' or .='IPv4ANY' or .='IPV4ANY' or .='ARP'])";
```

Comment:
According to the aclorch.cpp backend code, all fields and values are converted to upper case. I think we should convert that too before yang model's validation.